tox.ini

```diff
@@ -1,15 +1,18 @@
 [tox]
-envlist = py27
-
-[testenv:py27]
-deps =
-    mock
-    mozfile >= 1.1
-    mozprocess >= 0.22
-    pylama
-    pytest
-    wptserve >= 1.4.0
+envlist = py27, coverage
 
+[testenv]
+deps = -rrequirements/tests.txt
 commands =
     pylama mozdownload
-    py.test {posargs}
```

Comment:
It makes sense that they'd run twice if you just execute `tox` because it will run all the environments it finds. On Travis, `TOXENV` is set to `py27` so only that environment is run.